test/extended/images/tagimportmode.go

```diff
@@ -0,0 +1,191 @@
+package images
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/types"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	configv1 "github.com/openshift/api/config/v1"
+	imagev1 "github.com/openshift/api/image/v1"
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/library-go/pkg/image/imageutil"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/openshift/origin/test/extended/util/operator"
+)
+
+var _ = g.Describe("[sig-imageregistry][OCPFeatureGate:ImageStreamImportMode][Serial] ImageStream API", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLI("imagestream-api")
+
+	g.It("import mode should be PreserveOriginal or Legacy depending on desired.architecture field in the CV [apigroup:image.openshift.io]", func() {
+		TestImageStreamImportMode(g.GinkgoT(), oc)
```

Comment:
Given that the two test cases below cover both `Legacy` and `PreserveOriginal`, do we really need this test case? I can't really see what value it's adding, though I may be missing something.

Reply:
this test case is actually the only "necessary" one in my opinion because it changes the default import mode based on the CV's architecture. The other two test cases are specifically for testing the knob to change the import mode which I don't see being used frequently

Reply:
the main idea here is that we have these tests which are exercised behind the featuregate and once we let them run for a release (or maybe a few sprint cycles), we use that signal to promote the featuregate to default and then  I was thinking we remove the two tests which change the import mode knob.

Reply:
ack, thanks for expanding!